src/ml_tooling/baseclass.py

```diff
@@ -226,6 +227,7 @@ def make_prediction(
         data: Dataset,
         *args,
         proba: bool = False,
```

Comment:
Alternatively, proba could be a float and that's the threshold, if you can't have both `proba=False` and `threshold=0.2` - would probably have to rename it though

Reply:
I'll look into it